include/tvm/topi/reduction.h

```diff
@@ -345,6 +347,9 @@ inline Tensor collapse_sum(const Tensor& data, Array<PrimExpr> target_shape) {
       squeeze_axes.push_back(i_ax);
     } else if (oshape[o_ax] == 1) {
       --o_ax;
+    } else {
+      ICHECK(0) << "Shapes of collapse_sum do not match: data.shape = " << data->shape
```

Comment:
Let’s use `ICHECK(false)`
```suggestion
      ICHECK(false) << "Shapes of collapse_sum do not match: data.shape = " << data->shape
```